Tools/cases_generator/interpreter_definition.md

```diff
@@ -423,3 +443,19 @@ rather than popping and pushing, such that `LOAD_ATTR_SLOT` would look something
 
 From the instruction definitions we can generate the stack marking code used in `frame.set_lineno()`,
 and the tables for use by disassemblers.
+
+## Appendix A: Types
+
+The following types correspond to the following information:
```

Comment:
"information" is a bit vague.
Maybe say something like: "A value with any of the following annotations obeys the following predicate"
and expand the types to a predicate `&PyLong_Type` becomes `Py_TYPE(val) == &PyLong_Type` 
That way you can be more precise about whether it is an exact or subtyping relation.